src/index.html

```diff
@@ -4,13 +4,30 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,initial-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="/src/style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#" class="form-big"
+     data-qa="big">
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      action="#"
      class="form-big"
      data-qa="big"
    >
```